shell/platform/embedder/embedder.cc

```diff
@@ -712,6 +712,42 @@ inline flutter::PointerData::SignalKind ToPointerDataSignalKind(
   return flutter::PointerData::SignalKind::kNone;
 }
 
+// Returns the buttons for PointerData for the given buttons and change from a
+// FlutterPointerEvent.
+inline int64_t ToPointerDataButtons(int64_t buttons,
+                                    flutter::PointerData::Change change) {
+  switch (change) {
+    case flutter::PointerData::Change::kDown:
+    case flutter::PointerData::Change::kMove:
+    case flutter::PointerData::Change::kCancel:
```

Comment:
I don't think `kCancel` should always have non-zero buttons; it seems like it should be able to go either way. I'd expect it to be a no-op third case in the switch, with a comment explaining why.

Reply:
It's an oversight. Will fix.